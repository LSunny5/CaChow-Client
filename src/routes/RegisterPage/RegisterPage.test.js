import React from 'react';
import ReactDOM from 'react-dom';
import RegistrationPage from './RegisterPage';
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import { BrowserRouter } from 'react-router-dom';

describe(`RegistrationPage component`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        const props = {
            history: {
                push: () => { }
            }
        };
        ReactDOM.render(
            <BrowserRouter>
                <RegistrationPage {...props} />
            </BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the RegistrationPage route', () => {
        const wrapper = shallow(<RegistrationPage />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
});