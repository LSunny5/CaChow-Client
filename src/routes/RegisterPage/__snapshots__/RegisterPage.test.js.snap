// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationPage component renders the RegistrationPage route 1`] = `
<section
  className="formPage registerPage"
>
  <button
    className="backButton"
    onClick={[Function]}
  >
    <img
      alt="Go back"
      className="backArrow"
      src="/images/backarrow.png"
    />
    Back
  </button>
  <RegistrationForm
    onRegistrationSuccess={[Function]}
  />
</section>
`;
